README.md

```diff
@@ -19,146 +19,79 @@
-
-## Apps
-
-| Name                          | Description                            |
-| ----------------------------- | -------------------------------------- |
-| [`web`](./packages/web)       | The Audius web and desktop application |
-| [`mobile`](./packages/mobile) | The Audius mobile application          |
+For details on operating an Audius service, getting started with the Token and the API, see [docs.audius.org](https://docs.audius.org/).
 
 ## Packages
 
-| Name                                                      | Description                         |
-| --------------------------------------------------------- | ----------------------------------- |
-| [`stems`](./packages/stems)                               | The Audius client component library |
-| [`common`](./packages/common)                             | Shared code between web and mobile  |
-| [`eslint-config-audius`](./packages/eslint-config-audius) | Shared lint configuration           |
+| Name                                                                                            | Description                                                                                                                                                                           |
+| ----------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`commands`](./packages/commands)                                                               | CLI to perform actions against the protocol                                                                                                                                           |
+| [`common`](./packages/common)                                                                   | Shared code between web and mobile                                                                                                                                                    |
+| [`compose`](./packages/compose)                                                                 | Defines dependencies for audius-compose                                                                                                                                               |
+| [`contracts`](https://github.com/AudiusProject/audius-protocol/tree/main/contracts)             | The POA network smart contracts for the Audius protocol, encompassing user account, content listing, and content interaction functionality                                            |
+| [`creator-node`](mediorum)                                                                      | Maintains the availability of users' content via the Audius Storage Protocol, including user images and audio content. Also known as Content Node or mediorum.                        |
```

Comment:
Nit, I would say mediorum is the "content node protocol" and different from a content node, which is a registered chain-type, but no need to change here. what you've done is fine.  we need to work on this though.